PoGo.Necrobot.Window/Controls/PlayerInfo.xaml

```diff
@@ -110,9 +98,11 @@
                     </MultiBinding>
                 </TextBlock.Text>
             </TextBlock>
-
-
+            <TextBlock Grid.Column="2" Grid.Row="2" x:Name="tb_lukyegg_time" TextWrapping="Wrap" Text="{Binding Lucky_expires, FallbackValue=LuckyEggs: 00m 00s, StringFormat=LuckyEggs: \{0\}, UpdateSourceTrigger=PropertyChanged}" VerticalAlignment="Top" Width="158" Height="16"/>
+            <TextBlock Grid.Column="3" Grid.Row="2" x:Name="tb_incense_time" TextWrapping="Wrap" Text="{Binding Insence_expires, FallbackValue=Insence: 00m 00s, StringFormat=Insence: \{0\}, UpdateSourceTrigger=PropertyChanged}" VerticalAlignment="Top" Width="158" Height="16"/>
+            <Button x:Name="btnCollectPokeCoin" Click="GetPokeCoin_Click" Grid.Column="4" Grid.Row="1" HorizontalAlignment="Center" VerticalAlignment="Center" Margin="23.4,24.4,24,0.2" Grid.RowSpan="2" Width="111">
```

Comment:
I also dont see why 2 Grid Rows would be needed, unless pokecoins go over 9 Digits